modules/dnn/include/opencv2/dnn/dnn.hpp

```diff
@@ -1193,6 +1193,11 @@ CV__DNN_INLINE_NS_BEGIN
                                    const float sigma = 0.5,
                                    SoftNMSMethod method = SoftNMSMethod::SOFTNMS_GAUSSIAN);
 
+    /** @brief Apply softmax process to blob.
+     * @param[in] inblob The input blob.
+     * @param[out] outblob The output blob applied softmax.
+     */
+    CV_EXPORTS_W void softmax(InputArray inblob, OutputArray outblob);
```

Comment:
Please exclude adding of new public API from this PR - this requires extra discussion.

Reply:
I got it. Revert the last commit.